README.md

```diff
@@ -35,12 +35,12 @@ created/removed. Thus, these CI tests have been split up into three phases:
 
 ## Running these tests
 
-The setup and teardown scripts need to be passed the base-runtime mock configuration file, provided on the command line via the [avocado's parameter passing mechanism](http://avocado-framework.readthedocs.io/en/latest/WritingTests.html#accessing-test-parameters). Additionally, the smoke test script needs to be passed the path to the compiler test resource directory.
+The setup and teardown scripts may optionally be passed the base-runtime mock configuration file (default: resources/base-runtime-mock.cfg), provided on the command line via the [avocado's parameter passing mechanism](http://avocado-framework.readthedocs.io/en/latest/WritingTests.html#accessing-test-parameters). Additionally, the smoke test script may optionally be passed the path to the compiler test resource directory (default: resources/hello-world). Finally, all of the scripts may optionally be passed the name to use for the docker image (default: base-runtime-smoke).
 
 Run each of the phases in sequence. e.g.,
 
-    $ avocado run ./setup.py --mux-inject 'run:mockcfg:resources/base-runtime-mock.cfg'
-    $ avocado run ./smoke.py --mux-inject 'run:compiler-test-dir:resources/hello-world'
-    $ avocado run ./teardown.py --mux-inject 'run:mockcfg:resources/base-runtime-mock.cfg'
+    $ avocado run ./setup.py [ --mux-inject [ 'run:mockcfg:resources/base-runtime-mock.cfg' ] [ 'run:docker-image-name:base-runtime-smoke' ] ]
+    $ avocado run ./smoke.py [ --mux-inject [ 'run:compiler-test-dir:resources/hello-world' ] [ 'run:docker-image-name:base-runtime-smoke' ] ]
+    $ avocado run ./teardown.py [ --mux-inject [ 'run:mockcfg:resources/base-runtime-mock.cfg' ] [ 'run:docker-image-name:base-runtime-smoke' ] ]
 
```

Comment:
I'd suggest making this just use the defaults here and maybe create an alternate section describing optional config overrides.